### DISCUSSION TITLE
When will the compute time of my database be reset?

### DISCUSSION THREAD

**Discussion by lukaoes** [vercel/community#7065] [Feature Question]
### Question

I've been using Vercel Postgres for 27 days and have used 20 hours of compute time. Will it be reset after 3 days when I hit 30 days?

**Comment by lukaoes** [comment]
My friend has been using it for 31 days already but it still didn't reset. 

![image](https://github.com/vercel/community/assets/144058583/784ba21d-6ead-48d4-96b0-79917ddf00d2)

**Comment by LifeRIP** [comment]
Same question

![image](https://github.com/vercel/community/assets/89425013/d953a720-8513-48c2-ab06-12c882e1ed56)

**Comment by versecafe** [comment]
It should be reset every billing period, the created date is just when the DB was made and shouldn't reset unless you delete the whole DB

**Reply by gobbodev** [reply]
alright, but how do we know when the billing period will be, as a Hobby plan user?

**Reply by Benxter83** [reply]
hobby plan's compute time doesn't reset. if you reached 60hours you'd need to upgrade to pro or wait 30 days to use it again in hobby - but then you can't use your db.

**Reply by pawlean** [reply]
Cross-posting another discussion here: https://github.com/orgs/vercel/discussions/7349#discussioncomment-9935729

[a comment by an automated account is not shown]
